src/app/components/add-employee/add-employee.component.ts

```diff
@@ -15,8 +15,8 @@ export class AddEmployeeComponent implements OnInit {
     id: '',
     name: '',
     email: '',
-    phone: 0,
-    status: ''
+    phone: '0',
```

Comment:
these changes are because phone numbers would usually be represented as a string of characters and you would limit to be only numbers 

^[0-9*]$ is the simplest regex for validation